testing/run_tests.py

```diff
@@ -502,19 +502,13 @@ def make_test(name, flags=None, extra_env=None):
       xvfb.stop_virtual_x(build_name)
 
   if is_mac():
-    # flutter_desktop_darwin_unittests uses global state that isn't handled
-    # correctly by gtest-parallel.
```

Comment:
I double checked locally with gtest-parallel and my suspicion that this was just another instance of this problem was confirmed.  It runs fine after this change.